src/platform/user/profile/vet360/components/base/Vet360EditModalErrorMessage.jsx

```diff
@@ -58,6 +58,7 @@ export default function Vet360EditModalErrorMessage({
       isVisible
       onCloseAlert={clearErrors}
       scrollOnShow
+      scrollPosition="end"
```

Comment:
I think we might actually want this to be `start` since this alert is always at or near the top of whatever content it lives with. I realize that's a change in the existing functionality. I'll check this out locally to see how it works.

Reply:
yeah the current behavior, scrolling the Alert to the end of the viewport, isn't great :)

![scroll](https://user-images.githubusercontent.com/20728956/89066489-68462e00-d322-11ea-977c-079c44978a4c.gif)

Reply:
Awesome! I will remove the "end" 👍